### DISCUSSION TITLE
Migration maps

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/interactivity#21] [Cool Scrolly Ideas]
http://usmigrationflowmapper.com/

Potential ideas for this type of thingy... ways to modify:

 - SALURBAL L1/L2
 - Instead of State -> state/county level migration, we could do City level ? or City suburb or Congressional district?
 - migration stratafied by different populations
     - by race/ethncicity (not sure if census data will let use resolve this)
     - by sex 

Would love to get thoughts @usamabilal @alinasmahl1 @anujtanwar98

**Comment by usamabilal** [comment]
Oh yeah. YOu had already sketched a Shiny version of this at L1 level for SALURBAL right?
@her43 actually cleaned US migraiton data based on the IRS a long time ago...we must have it somewhere (for the US).

City to City, Rural to Urban, Urban to Rural (rarer) would all be great for the future. 

SALURBAL 2 may also include a lot more migraiton work, so we can use that there too.

**Reply by ran-codes** [reply]
Yea had it at L1 but now-where as feature rich and user friendly as this D3 version. 

Yea I think this may be a data issue. Census bureau has good data at the [county](https://www.census.gov/data/tables/2019/demo/geographic-mobility/county-to-county-migration-2015-2019.html), [metro](https://www.census.gov/data/tables/2019/demo/geographic-mobility/metro-to-metro-migration.html) and [state ](https://www.census.gov/data/tables/time-series/demo/geographic-mobility/state-to-state-migration.html) level **but not by race/ethnicty/sex and at below county level.**

**Reply by alinasmahl1** [reply]
ah I would love city to suburb too -- just to bring my dissertation work back in! 

the congressional district part would be somewhat intersting as well-- if we decide to keep pursuing that project Usama-- to thikn about distrcts changing andpeople changing districts (it's all bringing me back to gentrifcation work on context and composition changes)

**Reply by her43** [reply]
>  @her43 actually cleaned US migraiton data based on the IRS a long time ago...we must have it somewhere (for the US).
> 
> City to City, Rural to Urban, Urban to Rural (rarer) would all be great for the future.

The migration data I cleaned previously is in \\files.drexel.edu\colleges\SOPH\Shared\UHC\Projects\Bilal_DP5\Data\Environment\US\Migration\

My documentation:
[IRS Migration Protocol and Data Documentation.docx](https://github.com/Drexel-UHC/interactivity/files/11287491/IRS.Migration.Protocol.and.Data.Documentation.docx)

The lowest geographic level available in the data is county level. I'd also done CBSAs and Commuting Zones. Rural to urban and urban to rural would be easy.
